locales/en/plugin__odf-console.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
we don't need `locales/en/plugin__odf-console.json` `locales/en/public.json` `locales/en/translation.json` these three namespaces for translation...
we only need one `locales/en/plugin__dfr-console.json` (name can be chnaged later, but for now it is `plugin__dfr-console.json`)...

Reply:
all translations should be in `locales/en/plugin__dfr-console.json` file (just to make sure, this is not manual step... this should automatically happen when u run `yarn i18n`.... based upon `t(plugin__dfr-console.json~)`)

Reply:
removed the reference of odf but still, the translation file is getting generated when I am running the yarn i18n command.